cfme/common/provider_views.py

```diff
@@ -313,6 +314,7 @@ class ContainersProvidersView(ProvidersView):
      represents Main view displaying all Containers providers
     """
     table = Table(locator="//div[@id='list_grid']//table")
+    paginator = View.nested(PaginationPane)
```

Comment:
EntitiesView which is used in ProvidersView is a bit sophisticated. So, you probably didn't notice that paginator is already there. Nevertheless,  paginator is already present there. You don't need to add a new one.

https://github.com/izapolsk/integration_tests/blob/master/widgetastic_manageiq/__init__.py#L1980